lib/result-view.coffee

```diff
@@ -227,12 +226,9 @@ transformimeJupyter.consoleTextTransform.mimetype = [
 ]
 
 transform = transformime.createTransform [
-    transformimeJupyter.PDFTransform,
     transformime.ImageTransformer,
     transformimeJupyter.SVGTransform,
     transformimeJupyter.consoleTextTransform,
-    transformimeJupyter.LaTeXTransform,
     MarkdownTransform,
     transformime.HTMLTransformer,
-    transformimeJupyter.ScriptTransform
```

Comment:
I know. This is great!

I tried to include the transformers instead of removing, but they didn't work out of the box. So either way we will have to do some work on them in the future.
For example `PDFTransform` displays a link to the pdf, but atom doesn't allow it to be clicked.